File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/EnemyDragonKillBlinkingAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.DragonType;

import static com.bonepl.chromaleague.hud.PredefinedKeySets.BLACKWIDOW_FUNCTIONAL;

public abstract class EnemyDragonKillBlinkingAnimation extends AllyDragonKillBlinkingAnimation {

    public EnemyDragonKillBlinkingAnimation(DragonType dragonType) {
        super(BLACKWIDOW_FUNCTIONAL, dragonType);
    }
}

File: src/hud/animations/EnemyDragonKillBlinkingAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {BLACKWIDOW_FUNCTIONAL} from '../PredefinedKeySets.js';
import {AllyDragonKillBlinkingAnimation} from './AllyDragonKillBlinkingAnimation.js';

export class EnemyDragonKillBlinkingAnimation extends AllyDragonKillBlinkingAnimation {
    constructor(dragonType: DragonType) {
        super(BLACKWIDOW_FUNCTIONAL, dragonType);
    }
}
